owasp-top10-2017-apps/a6/stegonography/README.md

```diff
@@ -75,21 +77,43 @@ By visiting `http://localhost:10006/login` we get to the following screen:
     <img src="images/login_page.png"/>
 </p>
 
-A quick `admin:admin` guess revealed that we can successfully log in to the application and get to the admin's control-panel, as shown by the image bellow:
+### 🔥
+
+A quick `admin:admin` guess revealed that we can successfully log in to the application and get to the admin's control-panel, as shown by the image below:
 
 <p align="center">
     <img src="images/admin_page.png"/>
 </p>
 
+### 👀
+
 #### Verbose session token gives away unnecessary information
 
-After logging in to the application, it's possible to see it sets a session token: `jwtSessionToken`. As shown by the following image:
+After logging in to the application, it's possible to see it sets a session token: `nodejsSessionToken`. As shown by the following image:
 
 <p align="center">
-    <img src="images/jwt_token.png"/>
+    <img src="images/token.png"/>
 </p>
 
-Even though using verbose tokens might not pose a straight forward security vulnerability, on certain apps it might give away the hint an attacker needs to compromise it's security. 🕵️‍
+### 🔥
+
+Having a look at the token's name, we get a strong indication that the app might be running NodeJS. By using `searchsploit` an attacker could find a malicious code to exploit this vulnerability.
+
+To install this tool, simply type the following in your OSX terminal:
+
+```sh
+brew install exploitdb
+```
+
+Then, simply search for "NodeJS":
+
+```sh
+searchsploit nodejs
+```
+
+<p align="center">
+    <img src="images/available_exploits.png"/>
+</p>
 
```

Comment:
I missed one last paragraph closing up the entire attack narrative ideia, something like "Now the attacker has gained unauthorised access and knows the application is written in NodeJs".

Reply:
Thanks very much @Takehime! Great idea!